lib/check_statistics.rb

```diff
@@ -95,7 +95,7 @@ def number_of_whatsapp_conversations(team_id, start_date, end_date, type = 'all'
             all = 0
             user = 0
             business = 0
-            data['conversation_analytics']['data'][0]['data_points'].each do |data_point|
+            data.dig('conversation_analytics', 'data', 0, 'data_points')&.each do |data_point|
```

Comment:
@caiosba I suggest to sent a notification in case data_points is nil
i.e 
data_points = data.dig('conversation_analytics', 'data', 0, 'data_points')
send_notification if data_points.nil?

Reply:
OK! Let's add details to the ticket.

Reply:
@caiosba I see that the message that I should add will be close to the Sentry error so I keep raising error for nil data be removing `&.each do |data_point|` and iterate data without `&.`